lib/land/tracker.rb

```diff
@@ -240,6 +243,7 @@ def create_visit
       visit.referer_id    = referer.try(:id)
       visit.user_agent_id = user_agent.id
       visit.ip_address    = remote_ip
+      visit.raw_query_string = request.query_string
```

Comment:
@themoffatt How is this different than the normalized query string on `pageviews`?